### PR TITLE
Fix few breakout  room bugs.

### PR DESCRIPTION
**Pull request by Tainan404**
Fixes:
* Don't allow viewers users views all breakouts rooms meeting.
* fix Join modal being open when generate URL. #6787
* Fix users get stuck in `generating url...` state.

### CODE REVIEW

File: bigbluebutton-html5/imports/api/users/server/publishers.js [hunk 1]
```diff
@@ -57,11 +57,14 @@ function users(credentials, isModerator = false) {
...
-    selector.$or.push({
-      'breakoutProps.isBreakoutUser': true,
-      'breakoutProps.parentId': meetingId,
-      connectionStatus: 'online',
-    });
+    const User = Users.findOne({ userId: requesterUserId });
+    if (User.moderator) {
+      selector.$or.push({
+        'breakoutProps.isBreakoutUser': true,
+        'breakoutProps.parentId': meetingId,
```
Comment (antobinary) [inline bigbluebutton-html5/imports/api/users/server/publishers.js]: Should not these be in an $and condition, not an $or condition?

File: bigbluebutton-html5/imports/api/breakouts/server/publishers.js [hunk 2]
```diff
@@ -1,23 +1,27 @@
...
 
   if (moderator) {
-    const presenterSelector = {
-      $or: [
-        { parentMeetingId: meetingId },
-        { breakoutId: meetingId },
-      ],
-    };
+    const User = Users.findOne({ userId: requesterUserId });
+    if (User.moderator) {
```
Comment (antobinary) [inline bigbluebutton-html5/imports/api/breakouts/server/publishers.js]: I think we should add a check for User was found: `if (!!User && User.moderator) {...}` - in all three cases in this PR

File: bigbluebutton-html5/imports/api/users/server/publishers.js (hunk 1 above)
Comment (Tainan404) [reply bigbluebutton-html5/imports/api/users/server/publishers.js]: Nope, I wouldn't be able to see any, the first is to see the meeting users the second is to see user that joined in breakout and allow invite user that didn' join yet

**Comment by antobinary** [comment]
Now I am invited to join room first from main room, then from breakout... :)
Peek 2019-03-14 11-09

**Pull request merged**